nginx/requirements.txt

```diff
@@ -0,0 +1 @@
+# integration pip requirements
```

Comment:
We definitely won't. There will be a lot to clean up afterwards. But that's okay. (In our internal integration sdk board I have a column for "later")